Source/msg.cpp

```diff
@@ -226,8 +226,8 @@ BYTE *DeltaExportItem(BYTE *dst, TCmdPItem *src)
 	int i;
 
 	for (i = 0; i < MAXITEMS; i++) {
-		if (src->bCmd == 0xFF) {
-			*dst = 0xFF;
+		if (src->bCmd == UCHAR_MAX) {

```

Comment:
Do we know there is no enum for e.g. `MSG_NONE`? For squelch it makes more sense to use `UCHAR_MAX` since the value is truly numeric (i.e. decrementing by one has meaning). However for `bCmd` the variable is an enum, As such it makes more sense in my opinion to have an enum value for 0xFF, rather than using `UCHAR_MAX`.

Reply:
I don't think we have an enum that fits here. Also since it is always checked against the first byte of structure regardless if its `pD->x` or `pD->bCmd` I don't think an enum would be correct in any case.

Reply:
I might not know the exact context but:

> I don't think we have an enum that fits here. 

... why not extend [`enum _cmd_id`](https://github.com/diasurgical/devilution/blob/HEAD/enums.h#L2136) like this:
```
	...
	CMD_REMSHIELD         = 93,
	FAKE_CMD_SETID        = 94,
 	FAKE_CMD_DROPID       = 95,
 	NUM_CMDS              = 96,
+	CMD_NONE              = 0xFF,
 }
```

> Also since it is always checked against the first byte of structure regardless if its `pD->x` or `pD->bCmd` I don't think an enum would be correct in any case.

Do you mean any enum is always larger than one byte?

Reply:
Generally I don't think we add to enum as most of them are based on the psx data an so adding to them would change them from how the originals appeared. 

No, normally an enum would be associated with a specific parameter, but here it's more like the structure it self is being checked so and enum is probably not the right solution.

Reply:
Or we could just use `255` and be done with it...